.tito/packages/python-newdoc

```diff
@@ -1 +1 @@
-1.4.3-1 newdoc/
+1.5.0-1 newdoc/
```

Comment:
Just a thought: What are the costs of using a Makefile or m4 or something to have this is one place? I see the same number in at least 3 places...

Reply:
Good point. Fortunately, the `tito` build tool mostly takes care of this: I need to set the version manually in the `newdoc` command-line parser, and then tag a new version with `tito`, which updates the version in all the remaining places.

Still, I've forgotten to update the command-line parser a couple of times already.